src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -40,8 +40,8 @@ case class ClusterAlreadyExistsException(googleProject: GoogleProject, clusterNa
 case class InitializationFileException(googleProject: GoogleProject, clusterName: ClusterName, errorMessage: String)
   extends LeoException(s"Unable to process initialization files for ${googleProject.value}/${clusterName.string}. Returned message: $errorMessage", StatusCodes.Conflict)
 
-case class JupyterExtensionException(gcsUri: GcsPath)
-  extends LeoException(s"Jupyter extension URI is invalid or unparseable: ${gcsUri.toUri}", StatusCodes.BadRequest)
+case class BucketObjectException(gcsUri: GcsPath)
+  extends LeoException(s"The provided URI is invalid or unparseable: ${gcsUri.toUri}", StatusCodes.BadRequest)
```

Comment:
I made the error message more generic - is that alright?

Reply:
Yeah but you might want to say GCS URI just to be clear.